test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -296,6 +296,46 @@ function ExecuteWebRequest
     return $result
 }
 
+# Starts the ClientCertificateCheck listener
+function StartClientCertificateCheck 
+{
+    param([int]$Port)
+    $initTimeoutSeconds  = 15
+    $projectRootPath     = Resolve-Path (Join-Path $PSScriptRoot "\..\..\..\..\")
+    $appSrcPath          = Join-Path $projectRootPath  'test\tools\ClientCertificateCheck'
+    $appPublishPath      = Join-Path $appSrcPath 'bin'
+    $appDllPath          = Join-Path $appPublishPath 'ClientCertificateCheck.dll'
+    $serverPfxPath       = Join-Path $appSrcPath 'ServerCert.pfx'
+    $serverPfxPassword   = 'password'
+    $initCompleteMessage = 'Now listening on'
+    
+    $timeOut = (get-date).AddSeconds($initTimeoutSeconds)
+    $Job = Start-Job {
```

Comment:
@iSazonov  Ah. I see. Since it may be decided to move all tests to this, would it make sense to create a module of its own, similar to `HttpLitsener`? If we want to have logic detecting if it is running or not and with possibility of running multiple of these on different ports, it might warrant a `class` definition to contain the port and job for a specific listener and a module variable to track started listeners. It might also warrant a rename of the app to something more generic than `ClientCertificateCheck`

For this PR, to keep it small, Moving the start and stop functions to HelpersCommon and renaming them proper `Verb-Noun` would be a start.